libraries/AP_Motors/AP_MotorsMulticopter.h

```diff
@@ -28,6 +28,7 @@
 #define AP_MOTORS_THR_MIX_MIN_DEFAULT   0.1f    // minimum throttle mix
 #define AP_MOTORS_THR_MIX_MID_DEFAULT   0.5f    // manual throttle mix
 #define AP_MOTORS_THR_MIX_MAX_DEFAULT   0.5f    // maximum throttle mix default
+#define AP_MOTORS_THR_WARN_DEFAULT      -1      // output warning threshould (default: warning disabled)
```

Comment:
@hsteinhaus 
Can I sugest to use this parameter as the interval between messages ? i.e. if zero no warnings (current behavior) and positive would be the interval between warnings, up to a predetermined nr of seconds (300s ? 600s ?)